### DISCUSSION TITLE
<img :src="https://some.svg" class="fill-blue-500"> svg color not changing for external svg

### DISCUSSION THREAD

**Discussion by ArgekarStudios** [tailwindlabs/tailwindcss#6670] [Help]
fill not working for external SVG in IMG tag

**Comment by talkenson** [comment]
Hi! For example, when using raw svg - you can add `text-{current/color}` class, but this class won't be applied if you have a fill option in there. (Pic)
![image](https://user-images.githubusercontent.com/24260024/146870112-2c4d9927-81f4-4f71-bd66-124f90626b79.png)

My suggestion is to try using `text` class, or maybe you can provide your SVG

**Reply by ArgekarStudios** [reply]
Thanks for the suggestion..

**Comment by neupauer** [comment]
Hi @ArgekarStudios, if I'm not mistaken, this is not possible in HTML/CSS in general, so not even in Tailwind.

You need to use the `<svg>` tag to manipulate it with the utilities, as suggested by @talkenson.

**Reply by ArgekarStudios** [reply]
Thanks for the suggestion.
